easybuild/easyconfigs/g/GLib/GLib-2.54.2-foss-2017b.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'GLib'
+version = '2.54.2'
```

Comment:
The existing GLib 2.53.5 using `GCCcore/6.4.0` toolchain is probably sufficient?